example/lib/main.dart

```diff
@@ -24,10 +24,12 @@ class _MyAppState extends State<MyApp> {
         visualDensity: VisualDensity.adaptivePlatformDensity,
       ),
       home: Scaffold(
+        backgroundColor: Colors.grey,
         appBar: AppBar(
           title: Text(MyApp.title),
         ),
         bottomNavigationBar: SalomonBottomBar(
+          // backgroundColor: Colors.white,
```

Comment:
This is where we can showcase the adjustable background color

Reply:
The background color defaults to theme.bottomAppBarColor when no backgroundColor property is specified